resources/views/projects/create.blade.php

```diff
@@ -68,60 +59,58 @@
 			</div>
 		</div> --}}
 		<!-- Associated Course -->
-		<div class="field">
+		<div class="field {{ $errors->has('courses') ? 'error': '' }}">
 			<label for="courses">Associated Course</label>
 			<select name="courses" class="ui search dropdown">
 				<option value="">Web Development Class</option>
 				@if(isset($courses))
-					@foreach($courses -> all() as $course)
-						<option value={{$course['id']}}>
-						{{$course['name']}}</option>
+					@foreach($courses->all() as $course)
+						<option value={{ $course->id }}>{{ $course->name }}</option>
 					@endforeach
 				@endif
 			</select>
 		</div>
 		<!-- Labels -->
-		<div class="field">
+		<div class="field {{ $errors->has('label') ? 'error': '' }}">
 			<label for="label">Labels</label>
-			<select name="label[]" multiple="" class="ui fluid dropdown">
+			<select name="label" class="ui search dropdown">
 				<option value="">Finance</option>
 				@if(isset($labels))
 					@foreach($labels -> all() as $label)
-						<option value={{$label['id']}}>{{$label['name']}}</option>
+						<option value={{ $label->id }}>{{ $course->name }}</option>
```

Comment:
Copy + paste error. Should be `{{ $label->name }} `

`<option value={{ $label->id }}>{{ $label->name }}</option> `